swap_meet/clothing.py

```diff
@@ -1,2 +1,38 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    # May have to change fabric default values to None
+    def __init__(self, id=None, fabric="Unknown", condition=0):
```

Comment:
[opinion] I prefer spaces around `=` signs like `id = None` instead of `id=None`